Modules/Core/Common/include/itkFixedArray.h

```diff
@@ -324,7 +324,7 @@ class ITK_TEMPLATE_EXPORT FixedArray
     return m_InternalArray;
   }
 
-  ValueType *
+  [[nodiscard]] ValueType *
```

Comment:
Does PointType inherit this from FixedArray? Or does it already have `[[nodiscard]]` decorators?

Reply:
`itk::Point` inherits the `data()` member functions from `FixedArray` 👍